LICENSE.txt

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2018, conda-forge
```

Comment:
This file needs to be restored

Reply:
Done in 4e25437f8. Should I keep the LICENSE file in recipes/sycomore?

Reply:
It's included in the source tarball so you can use the filename from there